docs/dev/api_usage_examples/submit_listens.py

```diff
@@ -0,0 +1,53 @@
+'''
+submit_listens.py
+'''
+
+from common import AUTH_HEADER, EXAMPLE_RICK_TRACK, construct_url, ListenType
+
+import requests as req
+
+# The token in AUTH-HEADER must be the token of the user you're submitting listens for.
+def submit_listen(listen_type, payload):
+    """Submits listens for the track(s) in payload.
+
+    Args:
+        listen_type: One type from the ListenType enum.
+        payload: A list of Track dictionaries.
+
+    Returns:
+         The json response if there's an OK status.
+
+    Raises:
+         An HTTPError if there's a failure.
+         A ValueError is the JSON in the response is invalid.
+    """
+
+    url = construct_url("submit-listens")
+    json_data = {
+        "listen_type": listen_type.value,
+        "payload": payload
+    }
+
+    response = req.post(url, json=json_data, headers=AUTH_HEADER)
+
+    response.raise_for_status() # Will raise in case of a non-OK status
+
+    return response.json()
```

Comment:
I think an example of the data that is returned here would be good to add too.

Reply:
But this is `submit_listens.py`. If it all goes well, the only thing that's returned is `{"status": "ok"}`, which seems useless to include here.

Reply:
That is something that we could mention in `api_usage.rst`.

"If everything worked, you should have received a json response of the form `{"status": "ok"}`"

Reply:
Ah, I see. Will do.